controllers/controllers.js

```diff
@@ -68,6 +79,17 @@ exports.postComment = (req, res, next) => {
       })
 }
 
+exports.deleteComment = (req, res, next) => {
+    const comment_id = req.params.comment_id
+    Promise.all([selectCommentById(comment_id), removeComment(comment_id)])
```

Comment:
This does work but can be done using only the removeComment model as well.

Reply:
it's so invalid comment id's will be rejected otherwise i get a false positive. I'm not sure how to reject invalid id's any other way